automatic/360ts/360ts.nuspec

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>360ts</id>
+    <version>9.0.0.1146</version>
+    <title>360 Total Security</title>
+    <authors>Qihu 360 Software</authors>
+    <owners>Chocolatey</owners>
+    <licenseUrl>http://www.360totalsecurity.com/en/about.html</licenseUrl>
+    <projectUrl>http://360safe.com/</projectUrl>
+    <iconUrl> https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/b92d42be38598e16e7e526b3b8b43c3220439cd4/icons/360ts.png</iconUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>
+      A feature-packed software solution that provides users with a powerful antivirus, a junk cleaner and a system booster within the same interface
+    
+     ## Features
+      
+      - 360 Total Security offers your PC complete protection from Viruses, Trojans and other emerging threats.
+      - Whether you are shopping online, downloading files or chatting with your friends you can be sure that 360 Total Security is there to keep you safe and your computer optimized.
+      - Clean-up utility is just one click away to keep your PC in optimal condition.
+    
+    ## Notes
```

Comment:
The notes section is meant for notes regarding the package, or the installation/uninstallation of the software.
What you have listed here in the notes section, looks more like stuff that should be in the Features section.